index.d.ts

```diff
@@ -156,11 +156,11 @@ interface BlockSubscriptionCallbacks {
   onRemoved?: (block: Block) => void;
 }
 
-export enum OrderState {
-  All = "ALL",
-  Cancelled = "CANCELLED",
```

Comment:
This will be a larger if this drops the "L". if we want to do it, we should line it up with another DB forced resync

Reply:
Oh, but isn't AN using "OrderState.Canceled", not "OrderState.Cancelled"?

Reply:
We can, and should, do a migration to fix the DB instead of doing force re-syncs.